source/includes/markdown/_07-auth.md

```diff
@@ -161,18 +164,24 @@ scopes. An exhaustive list of the supported scopes is provided here:
 
 #### Request
 
-If your app received a code from the authorization endpoint, it can now be
-exchanged for a proper token, optionally including a *refresh_token*, which can
+When your app receives a code from the authorization endpoint, it can now be
+exchanged for a proper token. You may also get a `refresh_token`, which can 
```

Comment:
Hmm, this sounds to me on first read (in both the old and new version here) that we might send back a `refresh_token` on the first leg of the authorization flow - that is, through the browser - and this will only ever come back from the code exchange request.

How do you feel about omitting this and coving it below?

Specifically below where we say (Github won't let me comment out of the scope of this PR)

3. Exchange the code for a **token** via the token exchange endpoint

How do you feel about highlighting that this token is the `refresh_token` that's the *real* credential and should be stored, like:

3. Exchange the code for a **refresh_token** and, for convenience, an initial **access_token** via the token exchange endpoint.

4. When the short-lived `access_token` expires, the **`refresh_token`** can be used with the token exchange endpoint, without user intervention, to get a fresh `access_token`.